services/adgs/config/adgs_ws_config.yaml

```diff
@@ -101,7 +101,7 @@ adgs:
     free_text_search_operations:
       $filter:
         union: ' OR '
-        wrapper: '{}'
+        wrapper: '"{}"'
```

Comment:
no, it must be without quotes. I do the remark only once but there are more occurrences to fix